src/components/Modal/Modal.vue

```diff
@@ -735,15 +735,13 @@ export default {
 			max-width: 90%;
 			width: 600px;
 			max-height: 90%;
-			height: 600px;
```

Comment:
maybe we should give it a min-heigt of e.g. 300px now that we don't enforce a specific size anymore?

Reply:
What about a `width: clamp(300px, 600px, 90vw);`?

Reply:
Yeah right, off topic. I just stumbled upon this definition, because I prefer `clamb`, `min `and `max `over `min-width`, `max-width`, ... .

Reply:
okay, feel free to open a PR for discussion. For me using min, max, etc. is fine :)

Reply:
Ah, no. That's not valuable, because the result is the same. Just consumes time. For me this are just small  'by the way' improvements of the css code. I am more satified that I can remove my height hack after the release.

Reply:
okay 👍